api.ts

```diff
@@ -1619,6 +1628,8 @@ export interface DynamicSyncTableOptions<
    * in placeholderSchema will be rendered by default after the sync.
    */
   placeholderSchema?: SchemaT;
+
+  autocomplete?: CellAutocompleteMetadataFunction;
```

Comment:
Any reason the compiled property is called `cellAutocomplete` but this isn't?

Reply:
Renamed this to `propertyAutocomplete`